tests/integration/helpers/selectLocale.js

```diff
@@ -0,0 +1,11 @@
+import { t as testcafe, Selector } from "testcafe";
+
+export const selectLocale = async (locale) => {
+  const localeSelect = Selector("#locale > select");
+  const localeOption = localeSelect.find(`option[value="${locale}"]`);
+  await testcafe.click(localeSelect).click(localeOption);
+};
+
+export const selectLocaleFromDialog = async (locale) => {
+  await testcafe.click(`[data-test="languagePicker-${locale}"]`);
+};
```

Comment:
What is the difference between these two approaches?

Reply:
The first one is selecting locale from the menu dropdown. The second one selects locale from the DialogBox that opens up in a fresh session.

Reply:
The Dialog Box meaning the Language Picker